airbase-policy/src/main/resources/checkstyle/airbase-checks.xml

```diff
@@ -49,6 +49,10 @@
         <property name="format" value="^import static .*\.(of|copyOf|valueOf);$" />
         <property name="message" value="The following methods may not be statically imported: of, copyOf, valueOf" />
     </module>
+    <module name="RegexpSingleline">
+        <property name="format" value="^import static (?!java\.lang\.String\.format;).*\.format;" />
+        <property name="message" value="The format method cannot be statically imported" />
```

Comment:
How about making the message
```
Only 'format' from java.lang.String may be statically imported
```
This should give the user a better hint about what is wrong